### PR TITLE
Images::getInfoFromURL[Cached]() will both return empty arrays ...

### PR DESCRIPTION
**Pull request by Quix0r**
... on error and that needs to be reflected here, too.

Follow-up PR to https://github.com/friendica/friendica/pull/11673

### CODE REVIEW

File: statusnet/statusnet.php [hunk 1]
```diff
@@ -1050,7 +1050,7 @@ function statusnet_createpost(App $a, $uid, $post, $self, $create_user, $only_ex
 		$own_url = DI::pConfig()->get($uid, 'statusnet', 'own_url');
 
 		if ($content->user->id == $own_url) {
-			$self = DBA::selectFirst([], ['self' => true, 'uid' => $uid]);
+			$self = DBA::selectFirst('', ['self' => true, 'uid' => $uid]);
```
Comment (MrPetovan) [inline statusnet/statusnet.php]: This call is missing a table name. The empty array means `*` for the field selection parameter.

File: twitter/twitter.php [hunk 2]
```diff
@@ -1279,7 +1279,7 @@ function twitter_fix_avatar($avatar)
 	$new_avatar = str_replace("_normal.", "_400x400.", $avatar);
 
 	$info = Images::getInfoFromURLCached($new_avatar);
-	if (!$info) {
+	if (empty($info)) {
```
Comment (MrPetovan) [inline twitter/twitter.php]: `empty()` is superfluous as empty array is natively cast as false in a condition.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: statusnet/statusnet.php (hunk 1 above)
Comment (Quix0r) [reply statusnet/statusnet.php]: I added the `*` there.

File: twitter/twitter.php (hunk 2 above)
Comment (Quix0r) [reply twitter/twitter.php]: Once we have decided that in main repository's PR, I will change it here as well. :-)

File: statusnet/statusnet.php [hunk 3]
```diff
@@ -1050,7 +1050,7 @@ function statusnet_createpost(App $a, $uid, $post, $self, $create_user, $only_ex
 		$own_url = DI::pConfig()->get($uid, 'statusnet', 'own_url');
 
 		if ($content->user->id == $own_url) {
-			$self = DBA::selectFirst('', ['self' => true, 'uid' => $uid]);
+			$self = DBA::selectFirst('*', ['self' => true, 'uid' => $uid]);
```
Comment (MrPetovan) [inline statusnet/statusnet.php]: No, I mean, it needs a table name as first argument, and then an array of field names which can be empty, meaning `*`.
```suggestion
			$self = DBA::selectFirst('table', [], ['self' => true, 'uid' => $uid]);
```
Not sure which table it is though but it looks like `contact`.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**